iothub_client/tests/iothubtransport_mqtt_common_ut/iothubtransport_mqtt_common_ut.c

```diff
@@ -2102,23 +2116,20 @@ TEST_FUNCTION(IoTHubTransport_MQTT_Common_Destroy_One_Message_Ack_succeeds)
     IOTHUBTRANSPORT_CONFIG config ={ 0 };
     SetupIothubTransportConfig(&config, TEST_DEVICE_ID, TEST_DEVICE_KEY, TEST_IOTHUB_NAME, TEST_IOTHUB_SUFFIX, TEST_PROTOCOL_GATEWAY_HOSTNAME, NULL);
 
-    QOS_VALUE QosValue[] ={ DELIVER_AT_LEAST_ONCE };
-    SUBSCRIBE_ACK suback;
-    suback.packetId = 1234;
-    suback.qosCount = 1;
-    suback.qosReturn = QosValue;
-
     IOTHUB_MESSAGE_LIST message1;
     memset(&message1, 0, sizeof(IOTHUB_MESSAGE_LIST) );
     message1.messageHandle = TEST_IOTHUB_MSG_BYTEARRAY;
 
     DList_InsertTailList(config.waitingToSend, &(message1.entry));
-    TRANSPORT_LL_HANDLE handle = IoTHubTransport_MQTT_Common_Create(&config, get_IO_transport, &transport_cb_info, transport_cb_ctx);
-    g_fnMqttOperationCallback(TEST_MQTT_CLIENT_HANDLE, MQTT_CLIENT_ON_SUBSCRIBE_ACK, &suback, g_callbackCtx);
-    IoTHubTransport_MQTT_Common_DoWork(handle);
+
+    TRANSPORT_LL_HANDLE handle = setup_iothub_mqtt_connection(&config);
```

Comment:
>setup_iothub_mqtt_connection [](start = 33, length = 28)

This particular test appeared to have a different MQTT operation callback queued for the DoWork to process. Can we preserve that?